packages/cf-style-const/src/variables.js

```diff
@@ -97,5 +97,5 @@ export default {
   colorFacebookBlue: '#3b5998',
   colorMainBackground: '#e6e6e6',
   zIndexMax: 1000,
-  modalBoxShadow: '0 0 0 9999px rgba(0, 0, 0, .7)'
+  modalBackdropBackground: 'rgba(0, 0, 0, .7)'
```

Comment:
As a general rule we try to keep colors in variables so we can track all the different colors we use across the app and eventually standardize them.  I think we should put this `rgba` in [`variables.js`](https://github.com/cloudflare/cf-ui/blob/master/packages/cf-style-const/src/variables.js)